xml/snapper.xml

```diff
@@ -733,7 +733,7 @@ c..... /var/lib/rpm/Sigmd5</screen>
    </warning>
   </sect2>
 
-  <sect2 xml:id="sec.snapper.backup">
+  <sect2 xml:id="sec.snapper.undo.delete_file">
```

Comment:
why not "sec.snapper.restore_file"? "undo delete file" seems like a double negative.

actually, i think the whole "undo" wording in this chapter is not ideal.